### PR TITLE
add v1.22.0-alpha.0

### PR DESCRIPTION
**Pull request by xrstf**
**What this PR does / why we need it**:
Bootstrap 1.22.0

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: v1.22/Dockerfile.amd64 [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ARG VERSION
+ARG ARCH=amd64
+ARG CHECKSUM=091393bad069ee8bc31798e2c91b86cca32a9a16318fb8a373ae381a47b180987ae1adebefefe50c1bf144e17e7cd0af9f499aad1986b216f2485ff59514edbe
+
+RUN apk add curl && \
+    curl -Lo node.tar.gz https://dl.k8s.io/${VERSION}/kubernetes-node-linux-${ARCH}.tar.gz && \
+    echo "${CHECKSUM}  node.tar.gz" | sha512sum -c && \
+    tar xzf node.tar.gz kubernetes/node/bin/kubectl kubernetes/node/bin/kubelet
+
+FROM eu.gcr.io/k8s-artifacts-prod/build-image/debian-iptables:buster-v1.5.0
```
Comment (xmudrii) [inline v1.22/Dockerfile.amd64:13]: Nit: we should use `k8s.gcr.io`.

```suggestion
FROM k8s.gcr.io/build-image/debian-iptables:buster-v1.5.0
```

**Review by xmudrii — APPROVED** [review]
/lgtm

**Comment by xmudrii** [comment]
@xrstf Leaving it up to you to remove the hold. The comment is just a non-blocking nit, we can ignore it for now.

File: v1.22/Dockerfile.amd64 (hunk 1 above)
Comment (xrstf) [reply v1.22/Dockerfile.amd64:13]: Let's do that in a dedicated PR for all vesions

**Pull request merged**